help/availability.md

```diff
@@ -0,0 +1,36 @@
+Availability of submissions
+<<<<<<< HEAD
+===========================
+=======
+==================================================
+>>>>>>> be30559e08ffc4c96964d3c4c2771a792f94b6e3
```

Comment:
```suggestion
```

Reply:
Wasn't sure what the heading stuff was and if it was needed.
Suggest using at top of pages the format:
# Heading1